javascript/javascript1/week1/index.html

```diff
@@ -0,0 +1,37 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Javascript week1</title>
+
+</head>
+
+<body>
+    <h1>Age-Ify</h1>
+    <input type="number" id="birthYear" value="1991" placeholder="BirthYear">
+    <input type="number" id="futureYear" value="2023" placeholder="FutureYear">
+    <p id="calculatedAge"></p>
+    <script src="Age-ify.js"></script>
```

Comment:
this is not the right place to keep JS. PleaeRead where should we include JS in the HTML page.